src/Mvc/Mvc.ApiExplorer/src/EndpointMetadataApiDescriptionProvider.cs

```diff
@@ -180,21 +180,17 @@ private ApiDescription CreateApiDescription(RouteEndpoint routeEndpoint, string
         }
 
         // Determine the "requiredness" based on nullability, default value or if allowEmpty is set
```

Comment:
Comment seems wrong now

Reply:
I updated this to reflect that we're sourcing the info from the generators which get it from nullability/allowEmpty/etc.